SeoPresentationInterface.php

```diff
@@ -23,6 +24,16 @@
 interface SeoPresentationInterface
 {
     /**
+     * Gets the SeoMetadata based on the content that contains the content.
+     *
+     * @param object $content
+     *
+     * @throws Exception\InvalidArgumentException
+     * @return SeoMetadata
+     */
+    public function getSeoMetadata($content);
```

Comment:
remove the method from the interface and meke the phpcr provider typehint the concrete class rather than the interface. it is bound to the implementation anyways.